recipes/_freebsd.rb

```diff
@@ -21,4 +21,5 @@
   package 'devel/gmake'
   package 'devel/autoconf'
   package 'devel/m4'
+  package 'lang/gcc49' if node['os_version'].to_i <= 903_000
```

Comment:
Doesn't that require chef-sugar? I was just following the example here: https://github.com/chef-cookbooks/build-essential/blob/master/recipes/_rhel.rb#L32